_posts/2020-18-06-Triplestores-Onotologies-Reasoning.md

```diff
@@ -298,14 +309,14 @@ first is called _forward-chaining_ which is implemented by GraphDB. In
 this strategy, when each assertion, in this case each triple, is added
 to the Triplestore, the ontologies loaded will be consulted and any
 triples that would be created by the logical rules set by the ontology
-are created as are any other logical implications.  This means that
-all inferrencing is done at the time that the triple is added to the
-database.  The second is called _backwards-chaining_ which is
+are created as they are any other logical implications.  This means
```

Comment:
"as they are" > "as are"? Apologies if an erroneous correction on my part is behind this.